pkg/boot/catalog.go

```diff
@@ -32,8 +33,29 @@ func GetCatalog(
 				err,
 			)
 		}
+
 		environmentName := azureConfig.Environment.Name
-		for _, svc := range catalog.GetServices(environmentName) {
+		enableDRServices := catalogConfig.EnableDRServices
+		enableMigrationServices := catalogConfig.EnableMigrationServices
+		for _, svc := range catalog.GetServices() {
+			serviceTags := svc.GetTags()
+			tagsMap := map[string]bool{}
+			for _, t := range serviceTags {
+				tagsMap[t] = true
+			}
+			// Skip services which the cloud doesn't support
+			if !tagsMap[environmentName] {
+				continue
+			}
+			// Skip migration services if disabled
+			if !enableMigrationServices && tagsMap[service.MigrationTag] {
```

Comment:
How about ignore case here?

Reply:
Yes, but it is noticed the constant tag is in lower-case while OSBA's convention is first letter in upper case.

Reply:
Ah, thanks for reminding. I'll just fix the constant value.